client/scss/components/_button.scss

```diff
@@ -151,35 +147,20 @@
   }
 
   &.button-small.bicolor {
-    padding-inline-start: 3.5em;
+    padding-inline-start: theme('spacing.14');
 
     .icon-wrapper {
-      width: 2em;
+      width: theme('spacing.8');
     }
 
     &.button--icon .icon {
       @include svg-icon(0.9rem);
-      padding: 0.25em;
+      padding: theme('spacing.1');
     }
   }
 
-  // + input[type=submit],
-  // + input[type=reset],
-  // + input[type=button],
   + .button {
```

Comment:
Let's move the default 'button next to another button; styles near the top, maybe just under the transition stuff around line 27.